hnn_core/gui/gui.py

```diff
@@ -23,13 +24,31 @@
                              _read_legacy_params)
 from hnn_core.viz import plot_dipole
 from IPython.display import IFrame, display
-from ipywidgets import (HTML, Accordion, AppLayout, BoundedFloatText,
+from ipywidgets import (HTML, Accordion, AppLayout, Box, BoundedFloatText,
```

Comment:
the ipywidgets imports should actually happen above the hnn_core imports and there should be a single-line space between the hnn_core imports and ipywidgets imports